packages/pancake-profile-sdk/src/profile-sdk.ts

```diff
@@ -104,7 +104,13 @@ class PancakeProfileSdk {
       } = await this.profileContract.getTeamProfile(teamId);
       const staticTeamInfo = teamsList.find((staticTeam) => staticTeam.id === teamId);
 
-      return { ...staticTeamInfo, isJoinable, name: teamName, users: numberUsers, points: numberPoints };
+      return {
+        ...staticTeamInfo,
+        isJoinable,
+        name: teamName,
+        users: numberUsers.toNumber(),
```

Comment:
If it's safe to call `.toNumber()` on these, and they are indeed being returned as ethers BNs rather than numbers, nice one.

Reply:
Yeah, we do same in FE now.